sky/backends/cloud_vm_ray_backend.py

```diff
@@ -66,7 +66,7 @@
 
 # Timeout (seconds) for provision progress: if in this duration no new nodes
 # are launched, abort and failover.
-_NODES_LAUNCHING_PROGRESS_TIMEOUT = 90
+_NODES_LAUNCHING_PROGRESS_TIMEOUT = 120
```

Comment:
Can we only increase this number for Lambda, as that will increase the waiting time for the quota limit error for other clouds?

Reply:
Updated! I turned `_NODES_LAUNCHING_PROGRESS_TIMEOUT` into a dictionary.